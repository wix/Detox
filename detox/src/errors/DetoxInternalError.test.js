const DetoxInternalError = require('./DetoxInternalError');

describe('DetoxInternalError', () => {
  it('should append a Report-Issue hint', () => {
    expect(() => { throw new DetoxInternalError('Should do better'); })
      .toThrowErrorMatchingSnapshot();
  });

  it('.from(message) should wrap the error message ', () => {
    expect(DetoxInternalError.from('Should do better')).toMatchSnapshot();
  });
});
