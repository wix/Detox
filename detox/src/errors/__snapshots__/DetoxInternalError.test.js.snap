// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`DetoxInternalError .from(message) should wrap the error message  1`] = `
"DetoxInternalError: Should do better
Please report this issue on our GitHub tracker:
https://github.com/wix/Detox/issues"
`;

exports[`DetoxInternalError should append a Report-Issue hint 1`] = `
"Should do better
Please report this issue on our GitHub tracker:
https://github.com/wix/Detox/issues"
`;
